great_expectations/expectations/core/expect_column_values_to_not_match_like_pattern_list.py

```diff
@@ -63,6 +71,19 @@ class ExpectColumnValuesToNotMatchLikePatternList(ColumnMapExpectation):
         [expect_column_values_to_not_match_like_pattern](https://greatexpectations.io/expectations/expect_column_values_to_not_match_like_pattern)
     """
 
+    like_pattern_list: Union[List[str], EvaluationParameterDict]
+
+    @pydantic.validator("like_pattern_list")
+    def validate_like_pattern_list(
+        cls, like_pattern_list: list[str] | EvaluationParameterDict
+    ) -> list[str] | EvaluationParameterDict:
+        if len(like_pattern_list) < 1:
```

Comment:
I feel like most functions/classes that take lists are totally happy to have empty lists. Is this validation for the sake of validation? (I'm guessing there's some underlying reason why it matters, but just want to check)

Reply:
I think we may have tests that rely on this logic - I say let's keep the exact logic and come back if we need to tweak individual expectations